packages/docs/docs/lambda/lambdarolepermissions.md

```diff
@@ -0,0 +1,102 @@
+---
+image: /generated/articles-docs-lambda-lambda-role-permissions.png
```

Comment:
Need to generate a new image - delete this line and then run `node extract-articles.mjs` in packages/docs

Reply:
thanks for this, I did not know that there is a utility for that.